misc/dockertest.py

```diff
@@ -51,12 +50,13 @@
 # Assumes default docker config dir
 DEFAULT_DOCKER_CONFIG = os.path.expanduser("~/.docker")
 
+# Assumes the trust server will be run using compose if DOCKER_CONTENT_TRUST_SERVER is not specified
+TRUST_SERVER =  os.getenv('DOCKER_CONTENT_TRUST_SERVER', "https://notary-server:4443")
+
 # Assumes the test will be run with `python misc/dockertest.py` from
 # the root of the notary repo after binaries are built
-NOTARY_CLIENT = "bin/notary -c cmd/notary/config.json"
-
-# Assumes the trust server will be run using compose
-TRUST_SERVER = "https://notary-server:4443"
+# also overrides the notary server location if need be
+NOTARY_CLIENT = "bin/notary -c cmd/notary/config.json -s {0}".format(TRUST_SERVER)
```

Comment:
I think this may cause problems because the `config.json` has SSL certs for our testing server, which would fail if applied to the `TRUST_SERVER` - I think we may need to only conditionally pass `cmd/notary/config.json` :|  Apologies for the problems this is causing!